src/main/java/models/User.java

```diff
@@ -0,0 +1,46 @@
+package models;
+
+import Interface.InterfacePrintInformation;
+import services.AutoCreateID;
+
+public class User extends AutoCreateID implements InterfacePrintInformation {
```

Comment:
User is not an AutoCreateID. Rename "AutoCreateID" to some general entity name BaseEntity/BaseIdGeneratingEntity/RootModel etc.

Reply:
+